src/main/scala/mesosphere/marathon/core/instance/LocalVolume.scala

```diff
@@ -0,0 +1,48 @@
+package mesosphere.marathon
+package core.instance
+
+import com.fasterxml.uuid.{ EthernetAddress, Generators }
+import mesosphere.marathon.api.v2.json.Formats._
+import mesosphere.marathon.state.{ PathId, PersistentVolume, VolumeMount }
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class LocalVolume(id: LocalVolumeId, persistentVolume: PersistentVolume, mount: VolumeMount)
+
+case class LocalVolumeId(runSpecId: PathId, name: String, uuid: String) {
+  import LocalVolumeId._
```

Comment:
This is kind of violating our code culture. Why not simply `LocalVolumdId.delimiter`?

Reply:
It is as it was. I just moved this code to the new file. I will change it.